basiclibrary/src/main/java/basiclibrary/Library.java

```diff
@@ -44,11 +55,35 @@ static boolean containsDuplicates( String[] args){
     }
 
 
-//Calculating Averages: Write a method that accepts an array of integers and calculates and
-// returns the average of all the values in the array.
+    //Calculating Averages: Write a method that accepts an array of integers and calculates and
+    // returns the average of all the values in the array.
+    public static int calculatingAverage(int[] num){
```

Comment:
This is good, but if you want to improve it, return a `float` or a `double`.  When you test a floating point number, it looks like this: `assertEquals(expected, actual, delta);`  'delta' is just the margin of error (something like 0.01), since floating point numbers are almost always have a small margin of inaccuracy.